vpr/src/place/place.cpp

```diff
@@ -1836,7 +1841,7 @@ static void get_bb_from_scratch(ClusterNetId net_id, t_bb* coords, t_bb* num_on_
     auto& grid = device_ctx.grid;
 
     ClusterBlockId bnum = cluster_ctx.clb_nlist.net_driver_block(net_id);
-    pnum = cluster_ctx.clb_nlist.net_pin_physical_index(net_id, 0);
+    pnum = net_pin_tile_index(net_id, 0);
```

Comment:
Similarly, maybe call this `net_pin_to_tile_pin_index()`?

Reply:
Done